# AdvCiv-SAS reachable-history commit diff
# Email privacy: email-shaped addresses redacted from generated commit-history text.
# History metadata layout: 3 (history-segment label; title-only diff header; full message redirected to anonymized Git logs).
# Cache format: 2
# Cache policy: 8c87d6e4bd0f
# Practical commit count: 4587
# Commit: 8e92114e8b61cb95cd3274328a159f285544fa30
# History segment: AdvCiv-SAS branch history
# Parent used for diff: 73a6f2eff35a96e878b183e02e3dad973c5b2db6
# Title: add more units (docs only + image map view), add disclaimer notice and copyright notice todo write them is about this i mean etc i mean anyways, thanks,
# Full commit message/metadata: see _1_AdvCiv-SAS/git_logs/git_log_anonymized_email_003_AdvCiv-SAS.txt.
# History segments describe ancestry/log partition, not authorship: SASBranch includes SAS work plus later upstream AdvCiv commits merged/imported into this branch.
# Practical commit counts can repeat on divergent/merged history; the full Git SHA is the canonical unique commit identifier.
# Diff policy: preserve textual source/docs/config history; summarize known generated/binary/noisy or exceptionally huge file patches.
# Git textconv/external diff drivers are disabled; redundant document formats such as ODT/PDF are summarized rather than converted.

[CHANGE SUMMARY - NUMSTAT + RENAME/MODE SUMMARY]
7	0	Copyright_notice
7	0	Disclaimer_notice
-	-	_1_AdvCiv-SAS/Docs_And_Appendixes/0.43 military tree_modified.png
91	0	_1_AdvCiv-SAS/Docs_And_Appendixes/notes_about_unit_design_choices
18	18	_1_AdvCiv-SAS/Docs_And_Appendixes/notes_about_unit_imports_listing_only.txt
 create mode 100644 Copyright_notice
 create mode 100644 Disclaimer_notice

[PATCHES]

diff --git a/Copyright_notice b/Copyright_notice
new file mode 100644
index 000000000..28c3379ae
--- /dev/null
+++ b/Copyright_notice
@@ -0,0 +1,7 @@
+todo
+
+music links, external images about this etc i mean anyways etc i mean anyways, thanks,
+
+and note that i hope you enjoy the game though and is to protect myself also and that i don't necessarily find it too sensical (opposite of nonsensical) but is as is about this at least etc (delete later this specific about this etc note etc) (but that
+i wanted to write so is not hindrance but later will be, cruel to say but i mean i would want to clean it up tidy it up free my mind of it indeed in some other case maybe oe other type of eprosn may have said to me but i focus and deelte this extra other type of eprson note too later was not my intention to mention it that way i apologize etc even if is indirect still etc todo also add quote civ4 the proper way this time and about this delete all this type of notes etc later those i find aobut this etc
+i mean anyways, thanks,)

diff --git a/Disclaimer_notice b/Disclaimer_notice
new file mode 100644
index 000000000..22a7bf540
--- /dev/null
+++ b/Disclaimer_notice
@@ -0,0 +1,7 @@
+todo
+
+(note: todo add disclaimer i don't condone jihad and may not be held responsible as such)
+
+(note: disclaimer i don't condone zionism add disclaimer note, and will not be held responsible as such, add disclaimer page todo maybe too and copyright notice page bout music or other assets images or/and others links etc)
+
+note also about any other non mentionned thing mentionned general disclaimer too etc that it's for entertainment purposes or/and soem other generic type of sentence about this i mean etc i mean anyways, thanks,

diff --git a/_1_AdvCiv-SAS/Docs_And_Appendixes/0.43 military tree_modified.png b/_1_AdvCiv-SAS/Docs_And_Appendixes/0.43 military tree_modified.png
index d4414467d..f3a17286f 100644
Binary files a/_1_AdvCiv-SAS/Docs_And_Appendixes/0.43 military tree_modified.png and b/_1_AdvCiv-SAS/Docs_And_Appendixes/0.43 military tree_modified.png differ

diff --git a/_1_AdvCiv-SAS/Docs_And_Appendixes/notes_about_unit_design_choices b/_1_AdvCiv-SAS/Docs_And_Appendixes/notes_about_unit_design_choices
index 2423c3b16..ec4b77747 100644
--- a/_1_AdvCiv-SAS/Docs_And_Appendixes/notes_about_unit_design_choices
+++ b/_1_AdvCiv-SAS/Docs_And_Appendixes/notes_about_unit_design_choices
@@ -776,3 +776,94 @@ end of quote
 - for the early sniper, similarly, quote https://en.wikipedia.org/wiki/Sniper#Second_Boer_War + https://en.wikipedia.org/wiki/Sniper#World_War_I
 
 - for the modern sniper similarly, quote https://en.wikipedia.org/wiki/Sniper#World_War_II
+
+- about the camel gun unit (industrial era), about this etc, i have put it to defense rather than offense as it was,
+i don't have another model except the camel cavalry from c2c that doesn't seem realistic with its occidental-like
+outfit (seems like just a cavalry with a camel with some other strange c2c models but i am thankful they have them
+at least for the ones i took if not more and took many of them from them kindly shared by them about this etc anyways),
+also the rest is only based on my intuition but i don't think (feel? Is about this, at least or not at least, but focus
+etc) camels would be effective as attackers and be able to apply flanking or other strategies which i don't know a lot
+about but i assume would happen this way, etc, the speed of the horse cavalry would just outmatch it, but again did
+not check and is about this. Also against other gun units unlike in previous eras, the camel would start to fall behind
+vs the horse about this etc i think based on intuition but etc, still should be helpful and provide some defense value
+especially in the desert or similar terrains, may adjust bonuses based on terrains also for this unit (camel defender)
+etc as i intend to do to all/most units, and is about this etc. It is also more convenient that i don't have a camel offense model this way, using it as/at defense(der) etc.
+
+- for the horse industrial/imperial age defender (i say imperial because of how many big empires fought with royal guards
+and such from little i know or saw in anime and such, and on this topic: ) i as soon as i realized it, which is today xd
+but anyways, is probably safe to put xd here since not many will read but anyways, if at all but etc, i like to share but
+to protect myself i want also is about me and this at least etc but anyways maybe delete this later before merging this
+pr or not but, anyways, what i mean is i want badly, not to play with feelings trying to focus on the this game and its
+units and not mislead, but i overthink, maybe delete this before merging also maybe no one reads etc, still, about this
+etc, i want badly to add an imperial/royal horse defender like in rose of versailles the anime, would be so badass.. But
+anyways
+
+- make the "title" of the units more clear sometimes maybe and more dinstinct from their unit type, i thought intuitively
+it was the same but is not after reading things indirectly but about this i mean anyways, thanks, for example "rifle cavalry",
+"horse knight", for example maybe, etc, Oscar!!!! Is about this and i "auto" typed quick the !! number maybe should delete this
+etc or maybe not because maybe tedious or maybe because as is and leave it as that or/and other or not but etc focus not
+mislead or other and talking to myself only but even if not focus maybe and thanks for reading but well thanks even if not
+reading but protecting myself and my ego so not too eager to those who hurt me but thanks i mean, that being said focus
+and maybe delete this later, and not (purposely) holding a grudge, do whatever i feel but is about me i am talking but each
+is free or not, weird way to talk why would i need to say that if each is free, am i trying to control others, maybe, but
+intention is not coercive, so now i am brainwashing others, not good, well anyways, todo delete this later, give them their
+freedom and choice, but focus on myself too, hear them maybe but would not change outcome but still feelings are important,
+to be heard at least, if i have the capacity for such a "noble" thing indeed too, but focus on myself, thanks for reading,
+delete before merging this too maybe, etc, thanks, etc, i mean anyways, thanks, also it's cool that git keeps history, so one
+does not need to overburden themselves and can read later at any time, if they prefer, unlike in the past, still i am mostly
+talking to myself, and i like attention, but also calm and peace, how contradictory, i like peacefulness but also initiative,
+make sense of that maybe, but it does to me and is who i am too, but i shouldnt say this and let others appreciate whatever
+they feel or want or do or not of me is not for me to decide, why am i even saying this, i am not trying to educate, just
+releasing some feelings and thoughts, should pay more respect to the thought process of others, not trying or thinking i am
+arrogant or condescending, i have a lot to learn too(?) and can be very short tempered, but i like that side of me, with all
+that said delete before merge this pr but glad i said it, if none saw it, so be it, if some saw it and it gave them comfort,
+all good maybe, but protect my ego and not overshare to those who hurt me, but i wanted to say this at least and be thanful,
+so thanks, it is not charity, but being grateful, freely not out of duty, but now i make less and less sense, who cares about
+all that, just do what i feel or not if i want or not, not overstress myself, delete this later etc i mean anywyas, thanks
+for reading sorry if bother, maybe no one read or some did, don't (necessarily) want to know, but free to tell if want or not,
+but i don't like this way of saying, sounds coercive or/and manipulative, just wanted to mention the option, but is maybe
+needless, well thanks for reading, delete later etc, anyways, thanks, the internet is so beautiful, i can say all what i want
+to all the world, but yet no one reads/hears me, i feel so free.. wanted to say this thought i just had or feeling rather
+or both maybe about this etc, but no need to overshare, protect myself too, delete later, etc i mean anyways, thanks, not
+trying to do publicity for myself and personality or feels or other... focus, i don't like the way or tone of saying this too,
+sounds entitled or arrogant or condescending or insensitive, anyways, delete this later maybe too before merging this pr etc,
+thanks, but glad i said it all, anyways, about this at least, etc, (at least was not doing consciously but i don't know
+i like to express myself indeed maybe, anyways delete later, etc) (as someone may have rightfully highlighted to me that i am
+maybe gifted at communicating indeed, thanks for saying that) (but still delete this all later and focus, but thanks really)
+, should not delude myself or overthink, but in my mind is find to explore all and not restrict though, just be realist even
+though i don't like to, but do or not what i want or not etc and focus on present not on parallel past or future past
+universes if that (most likely not but idk) even exists, but anyways, i mean anyways, delete later etc i mean anyways,
+was just saying cause i wanted, not expecting anything in return, but free to do or not, but i don't like how coercive or/and
+manipulative htta sounds, anyways delete later etc, advance on my mod!!! Or/and other things etc i mean anyways, but thanks
+for making me think of these or being a part of this, still focus on myself or/and other times not just now or not as i do
+or not or want or not etc i mean anyways, thanks,
+
+- the transition from the superheavy tank (FCM 2C) of WW1 (tank defender i (quite) luckily indeed about this too (thinking
+of this (and thinking of thinking of this etc...) todo delete later etc)) to the mechanized infantry does not fit that well
+as they have quite different roles from the little i know, but it should be fine (enough) i think i mean anyways etc because
+mechanized infantry also served as anti personnel from the little i saw about this topic and learned and is about this etc,
+i would not want the upgrade to be a downgrade about this etc, will see how i balance mechanized infantry to make it valuable,
+taking historical accuracy in mind about this etc, maybe there's a way but is really about this (only) that i (was) speaking...
+todo delete this later (not to say it's exlcusive as in excluding other possibilities, but better not mislead, is this what
+this person meant? Still not overstressing myself, most likely better this way, but i don't know, still do what's best for me
+and not entertain (too much) such thoughts, but not disallowing me to feel them or/and consider them, is not arrogant, just
+self-preservation really, and trying to be rational(ist) even though i am not (so much) (or maybe a bit more than i think but
+i don't know)) but anyways etc, about not this when i said idk bro, it was an automatic sentence, i didn't put any specific
+intent in mind, just a way to be friendly and a bit cool too unconsciously maybe? At least was the result, may have been
+misunderstood though, but was a cool as in pleasurable as in rather to not hurt my ego etc enjoyable as in it makes me happy
+to think of, but also painful and hurting my ego too, as i was not this person i wanted to be, and it hurts me even today,
+so focus on myself, and not overstress, delete this before merging, not being bitter, i understand now it's maybe for the
+best, i am maybe selfish to say this again and again here, but is my life... hope others don't mind, delete this before
+merging, focus on me no need say all but glad/wanted say that all or bit of all delete later etc i mean anyways, thanks,
+
+- for the sniper 8 (modern), trying to match this picture of the wikipedia page of the sniper mentionned earlier about this
+etc (https://en.wikipedia.org/wiki/File:Pav-Stamp.jpg) even though a bit earlier than modern era but should do maybe about
+this etc, as for the sniper 7.. change after writing this, read below etc, thanks,
+
+- ended up reverting this logic as the infantry IJA of realism invictus fits well the sniper 7 (atomic) maybe rather, will adjust a newer maybe more modern sniper 8 instead if i want to do or do or not etc delete this later maybe etc too maybe etc
+
+- about the christian neo religious unit (special ops, see notes about unit imports for details, (and disclaimer notice
+at root of this docs etc, thanks, ), it conveniently and withotu me noticing it happened that special ops are common between
+the usa (fbi, nsa, etc from little i know about this etc) and russia (kgb etc from little i know about this etc), both of
+which are historically christian countries, so may fit the narrative if i may say about this here etc and not entitled todo
+delete this later etc quite well, about this etc i mean anyways, thanks,

diff --git a/_1_AdvCiv-SAS/Docs_And_Appendixes/notes_about_unit_imports_listing_only.txt b/_1_AdvCiv-SAS/Docs_And_Appendixes/notes_about_unit_imports_listing_only.txt
index ce3026475..5b139e473 100644
--- a/_1_AdvCiv-SAS/Docs_And_Appendixes/notes_about_unit_imports_listing_only.txt
+++ b/_1_AdvCiv-SAS/Docs_And_Appendixes/notes_about_unit_imports_listing_only.txt
@@ -15,6 +15,7 @@ pikeman (not useful about this i mean anyways, thanks)
 chariot (not accurate, should be lance or archer, not sword if i'm not mistaken i mean anyways, thanks,)
 swordsman (not as pretty or accurate than the bronze era equivalent i chose about this i mean anyways, thanks)
 catapult (not accurate enough or pretty enough i mean clear enough too to distinguish medieval and early catapult)
+cavalry (same reason as rifleman, see below, thanks,)
 rifleman (not immersive enough / art not good enough about this etc, in comparison to realism invictus's model in particular etc)
 infantry (too generic and i have many more specialized gun units about this i mean anyways etc i mean anyways, thanks,)
 
@@ -67,11 +68,11 @@ islam: mujahid (ansar) realism invictus
 neo religion military
 paganism: gibbon goetia (hero) _ heretic ffh2
 hinduism: cassay (hero) - hindu cavalry history rewritten
-judaism: ? _ zionist colon/infantry
+judaism: semi-modern infantry (tsahal) _ zionist colon/infantry 
 buddhism: monk _  free monk ffh2 (high evasion against guns maybe?)
 daoism: nightwatch _ ninja ffh2 (halve spying costs? Is it possible with current/existing XML options, else find other effect maybe? Etc)
-christianism:
-islam: ? _ terrorist / jihadist
+christianism: semi modern infantry (usa) _ special ops realism invictus (note that it fits fbi and such but also kgb and such, both christian in historical culture, so may fit well)
+islam: anti tank infantry _ "allahu akbar" (jihadist), give it city attack and collateral damage, but weak(er) stats otherwise, maybe some evade too, not politically correct but historically accurate maybe, a bit funny too even though not for all but is arrogant to say delete later i wanted to do this about this though i mean at least focus i really meant about this at least when i wrote this delete later etc
 
 
 
@@ -339,8 +340,9 @@ chariot 2: chariot archer c2c
 chariot lancers
 -
 chariot lancer 2: mursili 2 c2c todo icon from screenshot i mean anyways etc i mean anyways, thanks,
+-
+
 
-horseman: horseman from history rewritten
 
 horse archers
 -
@@ -354,8 +356,8 @@ horse lancers:
 -
 -
 -
-horse lancer 3: horseman alemannic realism invictus todo icon
-horse lancer 4: heavy cavalry c2c
+horse lancer 3: horseman alemannic _ horse lancer realism invictus todo icon
+horse lancer 4: heavy cavalry c2c _ medieval horse lancer knight
 -
 -
 
@@ -364,7 +366,7 @@ horse swords:
 -
 -
 -
-horse sword 4: -
+horse sword 4: - medieval horse sword knight
 -
 -
 
@@ -376,7 +378,7 @@ horse gun attackers:
 -
 -
 horse gun attacker 5: -
-horse gun attacker 6: -
+horse gun attacker 6: cavalry _ rifle cavalry realism invictus
 -
 -
 
@@ -387,7 +389,7 @@ horse gun defenders:
 -
 -
 horse gun defender 5: carabinier c2c
-horse gun defender 6: 
+horse gun defender 6: cavalry (argentinian) _ royal horse rifle guard (some name like this maybe etc + todo maybe change icon too etc)
 -
 -
 
@@ -438,7 +440,6 @@ camel gun offense:
 -
 -
 camel gun offense 5: camel cuirassier _ camel gunner from c2c
-camel gun offense 6: meharist _ camel cavalry c2c
 -
 
 
@@ -448,8 +449,7 @@ camel gun defense:
 -
 -
 camel gun defense 5: hajana (hero) _ camel patroller history rewritten todo change icon use the one of the camel trench cavalry (c2c) for example, or the icon of imajaghan (hero) (history rewritten), or edit myself but not guaranteed result etc
-camel gun defense 6:
--
+camel gun defense 6: meharist _ camel rifle guard c2c
 -
 
 
@@ -543,8 +543,8 @@ gun sniper units
 -
 gun sniper 5:  (unique) _ long range marksman history rewritten
 gun sniper 6: rifleman (british) _ - sharpshooter realism invictus
-gun sniper 7: early sniper
-gun sniper 8: modern sniper
+gun sniper 7: infantry IJA _ early sniper realism invictus
+gun sniper 8: semi modern infantry (south asia) _ modern sniper
 gun sniper 9: ?
 
 
@@ -621,7 +621,7 @@ tank defense
 -
 -
 -
-tank defense 7: ? or maybe none etc i mean anyways, thanks, about this etc i mean anyways, thanks,
+tank defense 7: superheavy tank (FCM 2C) - superheavy tank (FCM 2C) realism invictus
 tank defense 8: - _ mechanized infantry? Seems to fit quite well, may leave here about this etc.
 tank defense 9:
 
@@ -636,9 +636,9 @@ tank offense
 -
 -
 -
-tank defense 7: - _ early tank - (but same name as in realism invictus to also highlight it's an early one? Etc, As i got the idea too about this i mean anyways at least, i mean anyways? Etc. About this i mean anyways etc, thanks)
-tank defense 8: improved medium tank realism invictus
-tank defense 9: - _ robotic tank -
+tank offense 7: -
+tank offense 8: improved medium tank realism invictus
+tank offense 9: - _ robotic tank -

[PATCH COVERAGE]
patchFiles=5 embeddedFiles=5 omittedFiles=0 embeddedPatchBytes=18022
